app/views/api/v1/targets/index.json.jbuilder

```diff
@@ -0,0 +1,5 @@
+json.targets @targets.each do |target|
+  json.target do
+    json.partial! 'api/v1/targets/info', target: target
+  end
+end
```

Comment:
We should try to avoid using `each` in views. It will generate big performance issues rendering the partial for each record.

Something like this is better
```ruby
json.array! @targets, partial: 'api/v1/targets/info', as: :target
```